packages/app/src/components/AppBase/AppBase.tsx

```diff
@@ -64,6 +65,16 @@ const AppBase = () => {
             <Route path="/settings" element={<UserSettingsPage />} />
             <Route path="/catalog-graph" element={<CatalogGraphPage />} />
             <Route path="/learning-paths" element={<LearningPaths />} />
+            <Route path="/admin" element={<AdminPage />} />
+            {dynamicRoutes
```

Comment:
as you are handling the routes here for admin and below there is a map run on all dynamicRoutes again so can that be an issue?

https://github.com/janus-idp/backstage-showcase/pull/932/files#diff-4ecb4d92de7b045a6d54c1e7327fb3f2e41f34316fd87dfa01a629d08a93f971R78-R88

Reply:
This I have to revisit, as I figured I should be able to [just map these pages to the admin route](https://github.com/janus-idp/backstage-showcase/pull/932/files#diff-4ecb4d92de7b045a6d54c1e7327fb3f2e41f34316fd87dfa01a629d08a93f971R70) and then filter them from the other block (it would be before [this ](https://github.com/janus-idp/backstage-showcase/pull/932/files#diff-4ecb4d92de7b045a6d54c1e7327fb3f2e41f34316fd87dfa01a629d08a93f971L68)`map`)  where these are mounted.  But I was surprised to find it didn't work at the time, so yep this needs a closer look.  I might have just confused myself with my test plugin :smile: